Remove slim gem and convert .slim templates to .erb [37]

kljuni opened the pull request.

nmondollot commented:
Great work!

How did you do it? Did you use an automated script? 

Are there edge cases we should double check?
For instance I found a small regression on http://localhost:3000/fr/fontainebleau/levels

<img width="669" alt="Capture d’écran 2024-06-04 à 18 31 47" src="https://github.com/boolder-org/boolder-rails/assets/330823/c11280de-0a27-45c2-8834-b2fe6532fb2d">

Other than that it seems to work great 👍

kljuni commented:
Thanks! I used ChatGPT4o and it mostly did great, I checked each new view render with the "slim" one, and made some slight changes sometimes to adapt.

I fixed the omission of a class in _area_levels. I  remember this one because I removed an unused span element but consequently also the ml-1 class. Other than this I can't think of some other edge cases.

nmondollot commented:
AI for the win 😎 

It looks great, thank you!
